src/main/scala/com/microsoft/hyperspace/index/PathResolver.scala

```diff
@@ -68,27 +68,6 @@ private[hyperspace] class PathResolver(conf: SQLConf) {
     new Path(conf.getConfString(IndexConstants.INDEX_SYSTEM_PATH, defaultIndexesPath.toString))
   }
 
-  /**
-   * Get the Hyperspace index creation path.
-   *
-   * @return Hyperspace index creation path.
-   */
-  def indexCreationPath: Path = {
-    getOption(IndexConstants.INDEX_CREATION_PATH)
-      .map(new Path(_, IndexConstants.INDEXES_DIR))
-      .getOrElse(new Path(systemPath, IndexConstants.INDEXES_DIR))
-  }
-
-  /**
-   * Get the Hyperspace index search paths.
-   *
-   * @return Hyperspace index search paths.
-   */
-  def indexSearchPaths: Option[Seq[String]] = {
-    getOption(IndexConstants.INDEX_SEARCH_PATHS)
-      .map(_.split(',').toSeq)
```

Comment:
These are not currently not used. I will create a separate issue to implement this scenario.

Reply:
Created #243.